server/db.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/reviews', { useNewUrlParser: true });
+
+
+const db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => console.log('connected to DB'));
+
+let reviewSchema = new mongoose.Schema({
+  author: String,
+  rating: Number,
+  date: String,
+  review: {
+    tittle: String,
+    review: String },
+  popularity: Number
+});
+
+let Reviews = mongoose.model('Reviews', reviewSchema);
+
+module.exports = {
+  db: db,
+  Reviews: Reviews
+};
```

Comment:
Don't forget a newline at the end of ALL of your files.

Reply:
^ Might be an indication that you're not using the proper linter settings yet.

Reply:
Will go in and check my linter settings.